bench/ssz_ex_bench.exs

```diff
@@ -0,0 +1,36 @@
+alias LambdaEthereumConsensus.SszEx
+alias LambdaEthereumConsensus.Store.StateDb
+alias Types.BeaconState
+
+# To run this benchmarks, checkpoint-sync is needed.
```

Comment:
I would remove this comment, since we don't need checkpoint-sync, we just need to have a state